inference/models/starcoder.yaml

```diff
@@ -0,0 +1,23 @@
+port: 8000
+name: starcoder
+route_prefix: /starcoder
+precision: 'bf16'
```

Comment:
This attribute should be removed.

Reply:
ic, removed since it's no longer useful.